src/main/java/uk/gov/digital/justice/hmpps/sentenceplan/jpa/entity/ActionEntity.java

```diff
@@ -94,18 +79,13 @@ private void update(List<ActionOwner> owner, String ownerOther, String descripti
         this.ownerOther = ownerOther;
     }
 
-    public static ActionEntity updatePriority(ActionEntity actionEntity, int priority) {
```

Comment:
This method was silly, I've updated the priority code to remove the need for it.